core/tests/data/test_config.toml

```diff
@@ -1,6 +1,7 @@
-# Host and port of the current actor (operator or verifier)
+# Host, port and index of the current actor (operator or verifier)
```

Comment:
(operator, verifier, or watchtower)

Reply:
Fixed in https://github.com/chainwayxyz/clementine/pull/373/commits/c132f86a9d90874b3656208570e6b6f4f70fb102